src/transformers/pipelines/conversational.py

```diff
@@ -19,137 +19,120 @@ class Conversation:
-    to the [`ConversationalPipeline`]. This user input is either created when the class is instantiated, or by calling
-    `conversational_pipeline.append_response("input")` after a conversation turn.
+    inputs and generated model responses.
 
     Arguments:
-        text (`str`, *optional*):
-            The initial user input to start the conversation. If not provided, a user input needs to be provided
-            manually using the [`~Conversation.add_user_input`] method before the conversation can begin.
+        messages (Union[str, List[Dict[str, str]]], *optional*):
+            The initial messages to start the conversation, either a string, or a list of dicts containing "role" and
+            "content" keys. If a string is passed, it is interpreted as a single message with the "user" role.
         conversation_id (`uuid.UUID`, *optional*):
             Unique identifier for the conversation. If not provided, a random UUID4 id will be assigned to the
             conversation.
         past_user_inputs (`List[str]`, *optional*):
-            Eventual past history of the conversation of the user. You don't need to pass it manually if you use the
-            pipeline interactively but if you want to recreate history you need to set both `past_user_inputs` and
-            `generated_responses` with equal length lists of strings
+            This is a legacy argument that should not be used in new code. It is kept only for backward compatibility.
         generated_responses (`List[str]`, *optional*):
-            Eventual past history of the conversation of the model. You don't need to pass it manually if you use the
-            pipeline interactively but if you want to recreate history you need to set both `past_user_inputs` and
-            `generated_responses` with equal length lists of strings
+            This is a legacy argument that should not be used in new code. It is kept only for backward compatibility.
 
     Usage:
 
     ```python
     conversation = Conversation("Going to the movies tonight - any suggestions?")
-
-    # Steps usually performed by the model when generating a response:
-    # 1. Mark the user input as processed (moved to the history)
-    conversation.mark_processed()
-    # 2. Append a mode response
-    conversation.append_response("The Big lebowski.")
-
-    conversation.add_user_input("Is it good?")
+    conversation.add_message({"role": "assistant", "content": "The Big lebowski."})
+    conversation.add_message({"role": "user", "content": "Is it good?"})
     ```"""
 
     def __init__(
-        self, text: str = None, conversation_id: uuid.UUID = None, past_user_inputs=None, generated_responses=None
+        self,
+        messages: Union[str, List[Dict[str, str]]] = None,
+        conversation_id: uuid.UUID = None,
+        past_user_inputs=None,
+        generated_responses=None,
```

Comment:
should support `addition_deprecated_kwargs` as `text` is deprecated but should be kept for backward compatibility.

Reply:
Added!